src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -12,19 +13,17 @@ public FIFORhymer(IntArrayStack intArrayStack) {
 		super(intArrayStack);
```

Comment:
All rhymer constructors should take IntArrayStackInterface as a parameter (The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable.). Then it will be possible to do 2.3.1 easily.